src/parse/asp/targets.go

```diff
@@ -159,6 +163,43 @@ func populateTarget(s *scope, t *core.BuildTarget, args []pyObject) {
 	}
 }
 
+// addExtraHashData creates a suitable []byte of data that can be used by
+// ExtraHashData.  If a pyString is passed in, return the string itself.  If a
+// dict is passed in, sort the keys and concatenate the key and value and return
+// it as a response.
+func addExtraHashData(s *scope, name string, obj pyObject) []byte {
```

Comment:
do you still need this given discussion in #503?

Reply:
no, this is just a byproduct of basing this branch on `gh-503b`.  I'll post an update soon.